pydarn/utils/coordinates.py

```diff
@@ -153,14 +153,16 @@ def gate2geographic_location(stid: int, beam: int, height: float = None,
     # too much converting back and forth.
     boresight = np.radians(SuperDARNRadars.
                            radars[stid].hardware_info.boresight.physical)
+    bmoff = np.radians(SuperDARNRadars.
+                       radars[stid].hardware_info.boresight.electronic)
     radar_lat = np.radians(SuperDARNRadars.
                            radars[stid].hardware_info.geographic.lat)
     radar_lon = np.radians(SuperDARNRadars.
                            radars[stid].hardware_info.geographic.lon)
     # Some beam seperations are negative which changes how the coordinates wrap
     # we absolute to make it easier of fov-color for cartopy plotting
-    beam_sep = np.radians(abs(SuperDARNRadars.
-                          radars[stid].hardware_info.beam_separation))
+    beam_sep = np.radians(SuperDARNRadars.
+                          radars[stid].hardware_info.beam_separation)
     rxrise = SuperDARNRadars.radars[stid].hardware_info.rx_rise_time
 
     # TODO: fix after slant range change
```

Comment:
This if statement doesn't appear correct (which may explain the "TODO" note) - the `beam_edge` value should be zero if `center` is True (ie the current logic should be swapped).

Reply:
@carleyjmartin yes, there still tends to be a lot of confusion around whether `frang` corresponds to the front edge or center of the first range gate (I think it's the latter).  The convention for center vs edge of a beam in terms of azimuth should be more straightforward though:  the `center = True` case should return the azimuth to the center of the beam.  At least in the northern hemisphere, when `center = False` it should return the azimuth to the "left" / western" beam edge.  Currently, it appears the logic in the python code is reversed from what it should be.